esp/logging/loggingagent/cassandraloggingagent/cassandralogagent.cpp

```diff
@@ -180,6 +178,52 @@ void CCassandraLogAgent::createTable(const char *dbName, const char *tableName,
     executeSimpleStatement(createTableSt.str());
 }
 
+void CCassandraLogAgent::addField(CLogField& logField, const char* name, StringBuffer& value, StringBuffer& fields, StringBuffer& values)
```

Comment:
what is the method actually doing... adding a field, or inserting a field value?